docs/blockchain-basics/introduction.md

```diff
@@ -15,6 +15,42 @@ import NotificationBar from '../../src/components/docs/NotificationBar';
   </p>
 </NotificationBar>
 
-### Lorem ipsum
+# Introduction to decentralisation and cryptocurrencies
 
-Lorem ipsum
\ No newline at end of file
+## Context
+On **31st October, 2008**, under the "Satoshi Nakamoto" _pseudonym_, the Bitcoin whitepaper is discreetly released. In the title and the "abstract" paragraph, we can understand that the aim of _his_ invention is creating an "electronic cash system", a kind of currency, which is able to disconnect itself from any centralised entity.
+
+On **3rd January, 2009**, the first datas of this system are locked inside a _block_, called the "Genesis Block", which actually contains the first exchanges of the created electronic currency in "bitcoins". The Genesis Block also includes a message in English which is _The Times_ newspaper's main title of the same day:
+"_Chancellor on brink of second bailout for banks_".
+
+At these dates, the financial crisis strikes, it is then easy to reinforce the conclusion that Bitcoin was created mostly for **de-centralisation of money**. To achieve such an aim, Bitcoin uses various Cryptography's techniques and tricks. This explains why we refer to Bitcoin or new systems based on the same mechanics as **crypto-currencies**.
+
+## Terminology: An important note
+With time, the word "cryptocurrency" has been replaced by another more global and mainstream one, because other systems were created to lift Bitcoin's limitations, or propose another phylosophy: _Blockchain_.
+
+The exact word "blockchain" does not appear anywhere in Bitcoin's whitepaper but reflects the technique to arrange and chain datas for the purpose of validation and _immutability_ of exchanges. In Bitcoin's whitepaper, we can find what "blocks" or "chains" are, but they are "just" arranged in a _chain of blocks_.
+
+Hence, the polysemy that the reader would be advised to take into account with caution: "Blockchain" could refer the entire system; or the chained data-structure schema.
+
+## A brief history of the "Blockchain"
+If the newer systems rely directly on Bitcoin or implement their own similar mechanics anyway, Bitcoin itself relies on a lot of pretty old information technologies (IT), from as far as 1973, and until the Genesis Block from 2009. Bitcoin's system is actually around 36 years of research, trials and experiments, and failures.
+
+Needless to say Bitcoin relies on Internet, itself relying on _TCP/IP_ from 1974. But Bitcoin and newer systems also use _Cryptography_, which is, actually, as old as Ancient History. However, what we mean by "Cryptography" in these systems is _Modern Cryptography_, which is linked to first calculating machines and research in modern Mathematics. Very important researchs like for the Diffie, Hellman, and Merkle protocol, or RSA (_Rivest, Shamir, Adleman_) and Public-Key Cryptography, are both respectively from 1976 and 1977.
```

Comment:
Needless to say Bitcoin relies on Internet, itself relying on _TCP/IP_ from 1974.
Bad phrasing => Bitcoin relies on the Internet, which itself has being relying on _TCP/IP_ since 1974.